.github/renovate.json

```diff
@@ -19,6 +19,6 @@
       "automerge": true
     }
   ],
-  "schedule": ["after 2am on sunday"],
-  "prConcurrentLimit": 20
+  "schedule": ["after 2am on wednesday", "after 2am on sunday"],
+  "prConcurrentLimit": 10
```

Comment:
```suggestion
  "rebaseWhen": "never",
  "prConcurrentLimit": 10
```

Reply:
@sampankumar adding configuration for "rebaseWhen" to never - will save resources when a merge to master is triggered. As PRs from renovate will not all automatically rebuild

Reply:
Cool thanks @HarryH96